### ISSUE TITLE
After Devshop Ugrade the Status Report is still on 0.5.1 version

### ISSUE THREAD

**Issue by mecmartini** [opendevshop/devshop#37]
I launched 'devshop upgrade' on my devshop environment, and everything seems to be executed fine, except that, on the Status Report of devshop, it's still reported the 0.5.1 version, as you can look on this screen.

![screen-capture](https://cloud.githubusercontent.com/assets/2794938/11845395/001f8a52-a414-11e5-9256-fddbbbeaeaca.png)

Looking at the /var/aegir dir I can see only devmaster-0.6.0 as devmaster dir.

Following the aegir vhost of devshop domain:
```bash
<VirtualHost *:80>

  DocumentRoot /var/aegir/devmaster-0.6.0
  ServerName soulwebdev.com
  SetEnv db_type  mysqli
  SetEnv db_name  xxxx
  SetEnv db_user  xxxx
  SetEnv db_passwd  xxxx
  SetEnv db_host  localhost
  SetEnv db_port  3306


<IfModule mod_rewrite.c>
  RewriteEngine on
  RewriteRule ^/files/(.*)$ /sites/soulwebdev.com/files/$1 [L]
  RewriteCond /var/aegir/devmaster-0.6.0/sites/soulwebdev.com/files/robots.txt -f
  RewriteRule ^/robots.txt /sites/soulwebdev.com/files/robots.txt [L]
</IfModule>

# Extra configuration from modules:


# PROVISION_LOGS
# ==============
# Site-specific logs have not been enabled.
    # Error handler for Drupal > 4.6.7
    <Directory ~ "sites/.*/files">
      <Files *>
        SetHandler This_is_a_Drupal_security_line_do_not_remove
      </Files>
      Options None
      Options +FollowSymLinks

      # If we know how to do it safely, disable the PHP engine entirely.
      <IfModule mod_php5.c>
        php_flag engine off
      </IfModule>
    </Directory>

    # Prevent direct reading of files in the private dir.
    # This is for Drupal7 compatibility, which would normally drop
    # a .htaccess in those directories, but we explicitly ignore those
    <Directory "/var/aegir/devmaster-0.6.0/sites/soulwebdev.com/private/" >
      <Files *>
        SetHandler This_is_a_Drupal_security_line_do_not_remove
      </Files>
      Deny from all
      Options None
      Options +FollowSymLinks

      # If we know how to do it safely, disable the PHP engine entirely.
      <IfModule mod_php5.c>
        php_flag engine off
      </IfModule>
    </Directory>


</VirtualHost>
```
So which version I'm running?

**Comment by jonpugh** [comment]
Hi, @mecmartini,

Sorry for your problems. The issue is the version file.  I'm having trouble getting ansible to properly re-write this file on upgrade.  The Reports > Status page simply reads this file.

The version file is at /var/aegir/.devshop-version.  

You can tell you are on 0.6.0 because of your devmaster folder.   

You can check the version of your CLI by running 

```
devshop --version
```

If it is not 0.6.0, you can use git to checkout 0.6.0 in /usr/share/devshop

**Comment by jonpugh** [comment]
I recommend you manually change your version file (/var/aegir/.devshop-version) to 0.6.0

**Comment by mecmartini** [comment]
Ok, I manually changed the version in */var/aegir/.devshop-version* to *0.6.0*.

However, *'devshop --version'* still reported 0.5.1. As you suggested I used git checkout in */usr/share/devshop*: *'git checkout -b 0.6.0'*.

Now the 0.6.0 version is correctly recognized everywhere.

Thanks for the support, hope you can find a way to fix it!

**Comment by jonpugh** [comment]
Thanks for getting back!

I'm just about done eradicating the .devshop-version file for good, and creating a clean separation between CLI version and DevMaster version.

One of the original reasons for that file were that Drupal 6 install profiles have no version metadata.  OpenAtrium 6.x uses a VERSION.txt file to specify the version, so I am going to do that until we finish the move to drupal 7.

![screenshot from 2015-12-23 12 28 59](https://cloud.githubusercontent.com/assets/106420/11981790/e24959b2-a973-11e5-8e51-ddd5bc41a4a8.png)


@mecmartini I am concerned about the git command you ran, however:

> As you suggested I used git checkout in /usr/share/devshop: 'git checkout -b 0.6.0'.

With the -b flag, you created a new local branch from wherever you were in the tree.

0.6.0 is a tag in devshop, so you must delete your local 0.6.0 branch (`git branch -D 0.6.0`) , then `git fetch`, then `git checkout 0.6.0` (without the -b).

It should say something like "Checkout out 0.6.0... you are now in DETACHED HEAD".  This is ok. You are now locked to version 0.6.0.

The next version of devshop CLI will contain a self-update command to update the CLI, which you will do before you run the upgrade command.

Stay tuned!

**Comment by jonpugh** [comment]
This has been fully resolved since 0.7.x.  

Please feel free to re-open if you have the problem again.
